scipy/signal/_fir_filter_design.py

```diff
@@ -6,14 +6,16 @@
 
 import numpy as np
 from numpy.fft import irfft, fft, ifft
+from scipy.signal import get_window, firwin, freqz2
```

Comment:
perhaps we can fix the circular import by importing these functions from the files they are defined in, rather than the `scipy.signal` namespace?

Reply:
Thanks. I fixed it. I see that thare is another issue in the CI, I think it is infra issue rather then code. WDYT?